src/components/shapes/attributes.js

```diff
@@ -224,6 +225,50 @@ module.exports = templatedArray('shape', {
             '`config.editable` or `config.edits.shapePosition`.'
         ].join(' ')
     },
-
+    label: {
```

Comment:
IMHO `title` may be a better attribute name here. In future we may be able to add `arrayOk` `label`  container to allow shape with multiple labels e.g. per vertex text.

Reply:
I think `label` is better. We have many examples of each in the repo - ticks, pie slices, hover, slider steps get labels; the overall plot, axes, colorbars, legends get titles. Maybe the distinction is simple things without other text get labels, more complex items with other text get titles?

`arrayOk` is interesting - might be nice at some point. Another way to do it would be to make each attribute inside the container `arrayOk`, that way the other attributes could be shared or not, as desired.

Reply:
Alright. Let's keep `label` as is. I like the idea of having `arrayOk` option in future.